evpp/TcpServer.h

```diff
@@ -144,12 +144,12 @@ class TcpServerEventLoopTmpl {
     }
 
     // channel
-    const TSocketChannelPtr& addChannel(hio_t* io) {
```

Comment:
Why change the type of return value?

Reply:
I think RVO([return value optimization](http://en.wikipedia.org/wiki/Return_value_optimization)) will work in modern compilers